src/gnatss/loaders.py

```diff
@@ -170,38 +172,65 @@ def load_travel_times(
     return all_travel_times
 
 
-def load_roll_pitch_heading(files: List[str]) -> pd.DataFrame:
+def load_roll_pitch_heading(files: Union[List[str], str]) -> pd.DataFrame:
```

Comment:
```suggestion
def load_roll_pitch_heading(files: "List[str] | str") -> pd.DataFrame:
```

Reply:
The ` | ` syntax is only supported from Python 3.10 onwards. This would fail our Python 3.9 unittests ...
References:
[3.9 Union](https://docs.python.org/tr/3.9/library/typing.html#typing.Union)
[3.10 Union](https://docs.python.org/tr/3.10/library/typing.html#typing.Union)

Reply:
Gotcha! Okay.. I forget what we support in terms of Python